packages/vxrn/src/nativePlugin.ts

```diff
@@ -164,8 +164,12 @@ export function nativePlugin(options: {
                   return out
                 })
                 .join(';')
+              const magicString = new MagicString(code + '\n' + forceExports)
```

Comment:
Let's first create the magic string for code, and then inject the forceExports with a new line explicitly with the magic string instance itself, so we generate proper source maps.

Reply:
good call!